src/charm.py

```diff
@@ -257,6 +269,11 @@ def _get_available_mappers(self) -> List[str]:
             for schema_file in self._mappers_local_dir_path.iterdir()
         ]
 
+    @property
+    def _peers(self) -> Optional[Relation]:
```

Comment:
The PR description claims that the secret is stored in the peer relation databag but I do not see were this is done. All I see is that the secret ends up being created.

Reply:
You are correct. I thought that after all juju secrets are a better place to store this, but I forgot to change the description.

Still there is a weird bug when I run this code in the github runners, so I should place it to draft for now.